pkg/apis/v1alpha1/awsnodetemplate.go

```diff
@@ -35,6 +36,16 @@ type SecurityGroupStatus struct {
 	ID string `json:"id,omitempty"`
 }
 
+// AMI contains resolved AMI selector values utilized for node launch
+type AMIStatus struct {
+	// Id of the AMI
+	// +optional
+	ID string `json:"id,omitempty"`
+	// Requirements of the AMI to be utilized on an instance type
+	// +optional
+	Requirements []v1.NodeSelectorRequirement `json:"requirements,omitempty"`
```

Comment:
Can you post an example of what this looks like?

Reply:
```
 ami:
    - id: ami-07b3242213379
      requirements:
      - key: kubernetes.io/arch
        operator: In
        values:
        - amd64
    - id: ami-07b42342ae053
      requirements:
      - key: kubernetes.io/arch
        operator: In
        values:
        - amd64
    - id: ami-0534235287b3f
      requirements:
      - key: kubernetes.io/arch
        operator: In
        values:
        - arm64
```